index.html

```diff
@@ -34,7 +34,7 @@
 </p>
 
 {% highlight bash %}
-zypper install yast2-devtools rubygem-yast-rake git-core
+zypper install yast2-devtools ruby2.1-rubygem-yast-rake git-core
```

Comment:
This fix has the same problem than the original line - it does not work across all versions of openSUSE. It will work only for versions in which ruby2.1 is the default ruby interpreter.

Hopefully, this will work in all versions
```
zypper install yast2-devtools  git-core
ruby_version=`ruby -e "puts RbConfig::CONFIG['ruby_version']"`
zypper install -C "rubygem(ruby:$ruby_version:yast-rake)"
```